Core/GDCore/IDE/Events/EventsVariableReplacer.h

```diff
@@ -73,6 +73,7 @@ class GD_CORE_API EventsVariableReplacer
    */
   const gd::String targetGroupName;
   const VariablesRenamingChangesetNode &variablesRenamingChangesetRoot;
+  // TODO There is no reason de delete events. This dead code should be removed.
   const std::unordered_set<gd::String> &removedVariableNames;
```

Comment:
Should we remove this feature? I don't see any use-cases for it. I think that we you remove a variable you either want to replace it with something else for instance a variable from another scope, a function, a literal... so you want the formulas untouched and see the occurrences in red to know where to do the changes.